source/contracts/reporting/Universe.sol

```diff
@@ -442,25 +397,16 @@ contract Universe is DelegationTarget, Extractable, ITyped, Initializable, IUniv
         if (_currentFeeDivisor > Reporting.getMaximumReportingFeeDivisor()) {
             _currentFeeDivisor = Reporting.getMaximumReportingFeeDivisor();
         }
+        if (_currentFeeDivisor < Reporting.getMinimumReportingFeeDivisor()) {
+            _currentFeeDivisor = Reporting.getMinimumReportingFeeDivisor();
+        }
```

Comment:
Don't we have a `min` and `max` function on `uint256` that could be used here instead?  (same for other cases of taking the min or max above).

Reply:
We do! Will change